utils/eventtracking.js

```diff
@@ -9,47 +9,14 @@ const settings = require('./settings');
 const TRACKING_ENABLED_KEY = 'trackingEnabled';
 const TRACKING_SESSION_ID_KEY = 'trackingSessionId';
 
+const shouldOptInByDefault = () => {
+    const isGitPod = !!process.env.GITPOD_WORKSPACE_URL;
```

Comment:
Can we do just process.env.GITPOD_WORKSPACE_URL?

Reply:
in this case we can, however it's pretty commonly used for explicit cast to boolean: https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript

main difference is if somebody downstream uses outside of logical operation (e.g. `shouldOptInByDefault() == = true` or `shouldOptInByDefault().toString()` – this can fail in unexpected way if not coerced into boolean here

I'd prefer to keep it as is because we return value outside of function (and it is reasonable for caller to expect it is boolean)